airgun/views/settings.py

```diff
@@ -9,6 +10,8 @@
 
 class SettingsView(BaseLoggedInView, SearchableViewMixin):
     title = Text("//h1[text()='Settings']")
+    email_button = Button(id="test_mail_button")
```

Comment:
This button is available only under "Email" tab, so you can't put it in main class variable - the way you have it, it won't be accessible if any other tab is selected.

Something like that should do the trick:
```python
class SettingsView(BaseLoggedInView, SearchableViewMixin):

    @View.nested
    class email(SatTab):
        test_button = Button(id="test_mail_button")
```

In entity, you only need `view.email.test_button` - `SatTab` has magic to select tab before child widget is interacted with (see [widgetastic_patternfly](https://github.com/RedHatQE/widgetastic.patternfly/blob/master/src/widgetastic_patternfly/__init__.py#L602) for details).

Reply:
Fixed.